src/Containers/Microsoft.NET.Build.Containers/Tasks/CreateNewImage.cs

```diff
@@ -31,7 +31,19 @@ public sealed partial class CreateNewImage : Microsoft.Build.Utilities.Task, ICa
 
     public override bool Execute()
     {
-        return Task.Run(() => ExecuteAsync(_cancellationTokenSource.Token)).GetAwaiter().GetResult();
+        try
+        {
+            Task.Run(() => ExecuteAsync(_cancellationTokenSource.Token)).ContinueWith(t => t).GetAwaiter().GetResult();
+        }
+        catch (TaskCanceledException ex)
+        {
+            Log.LogWarningFromException(ex);
+        }
+        catch (OperationCanceledException ex)
```

Comment:
Seemed we could just catch `OperationCanceledException`, and maybe we could add a `when(_cancellationTokenSource.IsCancellationRequested)` exception filter to catch exception only when `_cancellationTokenSource` cancelled